### PR TITLE
Fetch artifact eggs from jenkins and install

### PR DESCRIPTION
**Pull request by anguss00**
~~~WIP: DO NOT MERGE~~~

@lwgordonimos that she blows

### CODE REVIEW

**Comment by lwgordonimos** [comment]
@mhidas this should handle both scenarios of vagrant using a local repo for aodncore/aodndata, and "not vagrant" using the wheel packages from jenkins build pipelines.

File: bin/vagrant-wrapper.sh [hunk 1]
```diff
@@ -175,13 +175,17 @@ declare -r PO_PIPELINE_VM_NAME=po_pipeline_2
...
+    declare -r AODNDATA_GIT_REPO=git@github.com:aodn/python-aodndata
 
     export VAGRANT_STATIC_IP=10.11.12.13
     export VAGRANT_MEMORY=8192
     export VAGRANT_MOUNT_OPTIONS='dmode=777,fmode=766'
 
     _clone_git_repo $GEOSERVER_GIT_REPO src/geoserver || return 1
     _clone_git_repo $DATA_SERVICES_GIT_REPO src/data-services || return 1
+    _clone_git_repo $AODNCORE_GIT_REPO src/data-services || return 1
+    _clone_git_repo $AODNDATA_GIT_REPO src/data-services || return 1
```
Comment (mhidas) [inline bin/vagrant-wrapper.sh]: Don't you want to clone these into `src/python-aodncore` and `src/python-aodndata` ?

File: private-sample/nodes/po_pipeline_2.json [hunk 2]
```diff
@@ -126,6 +126,7 @@
             "error_dir": "/vagrant/src/error",
             "data_dir": "/s3/imos-data",
             "graveyard_dir": "/vagrant/src/graveyard",
+            "processing_dir": "/vagrant/src/processing",
```
Comment (mhidas) [inline private-sample/nodes/po_pipeline_2.json]: This is already in there! (line 123)

Comment (lwgordonimos) [reply private-sample/nodes/po_pipeline_2.json]: Oops, note sure if that came from a weird merge or what, but have removed!

File: bin/vagrant-wrapper.sh (hunk 1 above)
Comment (lwgordonimos) [reply bin/vagrant-wrapper.sh]: Quite right, copy/paste error. Fixed.

**Pull request merged**